pyrokinetics/cgyro.py

```diff
@@ -151,16 +151,16 @@ def write(self, pyro, file_name, directory='.'):
 
             # Find BETA_STAR_SCALE from beta and p_prime
             if pyro.local_geometry_type == 'Miller':
-                beta_prime_scale = - miller.beta_prime / (local_species.a_lp * beta * (miller.Bunit / miller.B0) ** 2)
+                beta_prime_scale = - miller.beta_prime / (local_species.a_lp * beta * miller.bunit_over_b0 ** 2)
```

Comment:
Here is an example where changing only `B0` won't give you the behavior desired to only change `beta`  as `BETA_STAR_SCALE` will also change